termux.md

```diff
@@ -2,34 +2,126 @@
 # Use GramAddict without pc
 
 
-## Step 1: Download and Install Termux
+## Step 1a: If you don't have Termux installed
+
+You can download Termux from f-droid or the Google Play store. Get the f-droid version, since the Play store version does not support some features.  
 
 [Apk from f-droid](https://f-droid.org/it/packages/com.termux/)
-or
-[Official PlayStore](https://play.google.com/store/apps/details?id=com.termux)
+
+## Step 1b: If you have Termux installed
+
+To get this to work, we need to delete all pip packages and python
+
+```
+# Remove all python packages
+
+pip freeze > unins && pip uninstall -y -r unins && rm unins
+
+# Find the path of the cache for pip...
+
+pip cache dir
+
+# ... and clean it
+
+rm -rf /data/data/com.termux/files/home/.cache/pip
+
+# Delete python completely
+
+apt remove --purge python
+```
+
 
 ## Step 2: Install required packages
```

Comment:
we should divide the packages to be installed for the two types of installation: with and without report support
since the installation on cell phones takes a long time, the user should be able to decide

Reply:
Agree, I'm sorry that I didn't think of that